### ISSUE TITLE
Can't list all project

### ISSUE THREAD

**Issue by gz65555** [galacean/vscode-extension#13]
![image](https://github.com/galacean/vscode-extension/assets/6926263/f16b0d01-3c1b-4bb6-b194-0551950bdd42)

**Comment by Sway007** [comment]
yes, there is a pagination process in the list request, a "More" button need be added in next version
